Content.Goobstation.Shared/MartialArts/Events/SharedMartialArtsEvents.cs

```diff
@@ -0,0 +1,24 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Goobstation.Shared.MartialArts.Events;
+
+[Serializable, NetSerializable]
+public sealed class MartialArtSaying(LocId saying) : EntityEventArgs
+{
+    public LocId Saying = saying;
+};
+
+[Serializable, NetSerializable]
+public sealed class BurnSomeMunta : EntityEventArgs
+{
+    public NetEntity Target;
```

Comment:
These 2 events aren't supposed to be networked so [Serializeable, NetSerializeable] isn't needed. And so, you can replace NetEntity with EntityUid if you remove these attributes.